install.js

```diff
@@ -0,0 +1,8 @@
+// F5 -> npm start -> npm prestart -> this script.
+// Runs `npm install` only if needed. Otherwise skips (and saves time).
+
+const fs = require('fs')
+const { execSync } = require('child_process')
```

Comment:
What type is this?

Reply:
Not following the questions. The "type" is this: https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options. Is that what you meant?